### PR TITLE
HI and H2 data for comparison

### PR DESCRIPTION
**Pull request by wmfw23**
Added new data sets for HI and H2 fractions and mass functions

### CODE REVIEW

File: data/GalaxyH2Fractions/conversion/convertSaintonge2017.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import unyt
+import numpy as np
+import os
+import re
+import sys
+import itertools as it
+
+ORIGINAL_H = 0.7
+
+unitless = (unyt.Solar_Mass / unyt.Solar_Mass).simplify()
```
Comment (JBorrow) [inline data/GalaxyH2Fractions/conversion/convertSaintonge2017.py:12]: You can use unyt.dimensionless here?

**Comment by JBorrow** [comment]
Just the one question about dimensionless units, and then I'm happy to merge.

File: data/GalaxyH2Fractions/conversion/convertSaintonge2017.py (hunk 1 above)
Comment (wmfw23) [reply data/GalaxyH2Fractions/conversion/convertSaintonge2017.py]: I love to learn

**Comment by JBorrow** [comment]
Thanks James!

**Pull request merged**